test/file.rb

```diff
@@ -76,8 +80,13 @@
 end
 
 assert('File.realpath') do
-  usrbin = IO.popen("cd bin; /bin/pwd -P") { |f| f.read.chomp }
-  assert_equal usrbin, File.realpath("bin")
+  if File.const_defined?(:ALT_SEPARATOR) && File::ALT_SEPARATOR
```

Comment:
Ok, I have removed it (https://github.com/iij/mruby-io/commit/99e10b8dc44cd7e362f23706cc94a3e0ac840ed7).  Thanks.